### PR TITLE
Unittest for ticketmasterAPI

### PR DESCRIPTION
**Pull request by Saphirkitoko**
Added the test unit that tests the functionality of ticketmaster_api. Please review!

### CODE REVIEW

File: tests/test_ticketmaster.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import sys
+import os
+
+# Add the 'apis' directory to sys.path so Python can find it
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'apis')))
```
Comment (HyungnaLim) [inline tests/test_ticketmaster.py:5]: I don't think you need this line

File: tests/test_ticketmaster.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# import TicketmasterAPI and TicketmasterEvent as normal
+from ticketmaster_api import TicketmasterAPI, TicketmasterEvent # type: ignore
```
Comment (HyungnaLim) [inline tests/test_ticketmaster.py:8]: You can just specify directory with period in front of the module name like below:
from apis.ticketmaster_api import TicketmasterAPI, TicketmasterEvent

**Comment by HyungnaLim** [comment]
All 3 tests passes, but somehow I get error:
ERROR:root:HTTP error
requests.exceptions.HTTPError: HTTP error

Also on line11, `TicketmasterEvent` is imported but not used. maybe consider remove it if you're not going to include it?

**Comment by rwalk651** [comment]
@HyungnaLim The test throws that error

@Saphirkitoko Looks good

**Review by rwalk651 — APPROVED** [review]

**Pull request merged**